packages/edit-post/src/components/header/header-toolbar/style.scss

```diff
@@ -20,6 +20,15 @@
 			display: flex;
 		}
 	}
+	&.show-icon-labels {
+		button[aria-label] {
+			display: block;
+			&::after {
+				display: block;
+				content: attr(aria-label);
```

Comment:
This is an interesting approach. I like it keeps the components relatively clean.

Reply:
I'd like to get some A11y feedback on this, as support for `content` in CSS differs a bit among assistive technologies. We don't wanna end up with a screen reader reading the label out loud twice.

Reply:
Indeed.

Reply:
I found that @afercia already had an opinion on this: https://github.com/WordPress/gutenberg/issues/10524#issuecomment-482079618
Will change the approach to meet a11y requirements.

Reply:
@swissspidy thanks Pascal for raising this concern. You're right that some screen readers read out CSS generated content but in this case the whole content of the buttons is overridden by the aria-label so that only the aria-label is announced. To be 100% sure, I tested it also with NVDA and JAWS on Windows Firefox / Chrome.

Reply:
Wouldn't it be easier to inline this label inside HTML when `showIconLabels` prop is set to true?